platform/mellanox/mlnx-platform-api/sonic_platform/sfp.py

```diff
@@ -1493,9 +1491,12 @@ def mgmt_phy_mod_pwr_attr_get(cls, power_attr_type, sdk_handle, sdk_index):
         sx_mgmt_phy_mod_pwr_attr = sx_mgmt_phy_mod_pwr_attr_t()
```

Comment:
So this struct name not changed with the new function?

Reply:
No.